web/src/stories/index.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react'; // eslint-disable-line import/no-extraneous-dependencies
+import { action } from '@storybook/addon-actions'; // eslint-disable-line import/no-extraneous-dependencies
```

Comment:
Should we have an explicit dependency on `@storybook/addon-actions`?

Reply:
maybe :)